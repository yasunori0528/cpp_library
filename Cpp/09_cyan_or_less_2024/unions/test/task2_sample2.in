20 11
10 3 4 2 2 3 5 2 4 5 3
83 45 92 24 94 22 52 23 79 72 49
3 4 5 9 10 11 12 13 18 20
6 9 3
5 7 9 11
5 12
1 10
7 2 16
6 9 10 15 14
1 13
6 13 15 17
8 7 12 4 15
2 8 19
