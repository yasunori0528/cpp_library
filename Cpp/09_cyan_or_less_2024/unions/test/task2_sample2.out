196
106
106
106
102
174
174
106
94
106
106
23
146
102
196
102
106
223
106
